app/serializers/user_serializer.rb

```diff
@@ -18,6 +18,8 @@ class UserSerializer
 
   preload :avatar
 
+  cache_total_count true
```

Comment:
should this provide an option to set the expires in time, e.g. for users it can be default from cache store or 1 min, if we use this on other models we may want to have a shorter configurable expiry time...?